content/tutorial/14-converters.md

```diff
@@ -47,18 +47,16 @@ There are multiple errors that this command may raise which you'll want to handl
 
 This command quite simply takes a member and a reason and sends them a message. Note here that we can just immediately access the member's `send` method, since the converter guarantees we either get a `Member` object, or an exception is raised if conversion fails. In this command, `member` will **never** be an object of any type other than `Member`.
 
-Try making a few of your own commands use converters, if you don't already! A few converters of interest to you might be:
-- `discord.TextChannel`
-- `discord.Colour`
-- `discord.Role`
+Try making a few of your commands use converters, if you don't already! A few converters of interest to you might be:
```

Comment:
Awesome job adding the docs links! Again I'd rather keep the `own` here for my writing style, though